terraform/workspace-variables/review.tfvars.json

```diff
@@ -25,5 +25,7 @@
   "deploy_azure_backing_services": true,
   "enable_monitoring": false,
   "enable_postgres_ssl": true,
-  "add_database_name_suffix": true
-}
+  "add_database_name_suffix": true,
+  "sidekiq_replicas" : 2,
```

Comment:
Please configure the same values are paas environments, such as https://github.com/DFE-Digital/teaching-vacancies/blob/main/terraform/workspace-variables/production.tfvars.json#L28-L29